### ISSUE TITLE
more details for `man_made=surveillance` & `surveillance:type=camera`

### ISSUE THREAD

**Issue by chrt223** [streetcomplete/StreetComplete#6009]
Follow-up https://github.com/streetcomplete/StreetComplete/issues/2816

### General
Affected tag(s) to be modified/added: [surveillance:*](https://wiki.openstreetmap.org/wiki/Key:surveillance)
Question asked:
- [surveillance:*](https://wiki.openstreetmap.org/wiki/Key:surveillance) What kind of surveillance is this?
- [camera:type](https://wiki.openstreetmap.org/wiki/Key:camera:type)  Type of camera, e.g. fixed, panning, dome, ALPR
- [camera:direction](https://wiki.openstreetmap.org/wiki/Key:camera:direction) The orientation of the camera

Possibly (if not overwhelming):
- [camera:mount](https://wiki.openstreetmap.org/wiki/Key:camera:mount)  How or where is the camera mounted/fixed?
- [surveillance:zone](https://wiki.openstreetmap.org/wiki/Key:surveillance:zone) Sometimes ambiguous (town/parking/traffic/shop/bank/building/entrance/nesting_site)
- https://wiki.openstreetmap.org/wiki/Key:ele (but many confusion possible with sea-level vs ground-level)

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose ¹
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort)

¹ **Useful**: It probably is: There are dozen of projects on this specific topic with new ones appearing every year. Here to name a fewrelated to privacy  or traffic/circulation:
- https://sunders.uber.space
- https://deflock.me/
- https://cctv.masspirates.org/
- http://osmcamera.dihe.de/
- https://www.banfacialrecognition.com/map/
- https://otc.armchairresearch.org/

**Comment by matkoniecz** [comment]
see https://github.com/streetcomplete/StreetComplete/issues/87#issuecomment-583740033 and https://github.com/streetcomplete/StreetComplete/issues/87#issuecomment-583879232 for what was discussed in past

**Comment by chrt223** [comment]
> How we can skip/tag cameras in locations inaccessible to general public? [...] collecting data about surveillance for private property [...]

I agree with others in that it's not a blocker because the description of [Key:surveillance](https://wiki.openstreetmap.org/wiki/Key:surveillance) is quite clear about objects:

> - **indoor** 	Surveillance inside enclosed buildings, for example inside a [museum](https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dmuseum), a [shop](https://wiki.openstreetmap.org/wiki/Key:shop), a [bank](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbank), or at a [fuel station](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dfuel)

=>  The vast majority of these are going to be either public places/services or shops/services opened to the public. [OSM guidelines regarding private property apply](https://wiki.openstreetmap.org/wiki/Limitations_on_mapping_private_information) and the the tag purpose and examples given are clear.

> - **outdoor** 	Surveillance of outdoor areas or outside enclosed buildings, for example on courtyards, at fuel stations, on parking areas, at entrances.

=> It's not insane to assume these nodes were added because they record public activities in the first place or are fixed in a place of public passage

> - **public** 	Surveillance in publicly accessible areas, for example on public places, generally controlled or delegated by some governmental organization, like on marketplaces, in [pedestrian zones](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dpedestrian) or at corners (often mounted on utility poles or masts).

=> Public by definition

https://github.com/streetcomplete/StreetComplete/issues/87#issuecomment-583740033 rightfully reminded that some quests already request information about items existing on private property what sounds generally acceptable as long as it's stay an exception (if a surveyor has no judgement, he shall not use the app in the first place)

About data collection, it's pretty clear that it's not about _private property_ but about public space surveillance. How many  `indoor` camera are exclusively recording private indoor movements except some of these with `surveillance:zone=building`. If such a case happened, then these items **must** be given [access](https://wiki.openstreetmap.org/wiki/Key:access)=[private](https://wiki.openstreetmap.org/wiki/Tag:access%3Dprivate)


So, because:
- We should assume general OSM guidelines for private property applied when adding nodes
- It's unlikely to find truly private camera given the documentation/usage examples of the tags
- In the event of cameras record exclusively private domain, these should have been tagged `access=private` in the first place
- Even GoogleStreetView provide access to many `public` cameras (private owners recording street)

I suggest to filter out:
**`( access=private || (surveillance=indoor || surveillance=<unset>) && surveillance:zone=building) )`**

@westnordost also mentioned:
> "here is a camera, is it inside or outside?" because the user might then not even know which camera is meant - if it is not even known where the camera is supposed to be

This is a theoretically valid concern for **indoor** camera **or** if an indoor/outdoor camera was placed nearby a public one but it's an edge-case:

Are indoor/outdoor not accessible nor visible from public places? As commented in the [Discussion page](https://wiki.openstreetmap.org/wiki/Talk:Key:surveillance:zone), cameras are placed where they can  broadly see/record what generally includes public space if possible.

Regarding the case of a [surveillance:type](https://wiki.openstreetmap.org/wiki/Key:surveillance:type)=camera **without** a `surveillance` tags, it's definitely a confusing/incomplete node (which are the ones StreetComplete is aimed to... complete with the help of surveyors' judgement)
(And if humans are needed to fix the data, it's safe to assume they have the basic understanding of what is public/private/visible/invisible and identifying the placement of a couple of cameras :))

**Comment by aspensmonster** [comment]
I have been using StreetComplete as a starting point for adding various Flock Safety devices --- principally, their Automated License Plate Readers (ALPRs) and shot spotters (gunshot detectors) --- as well as ALPRs by other vendors (Axis Communications) and dome cameras. I definitely miss the ease of a fully fleshed out set of quests, as I have to go back in via iD to flesh out the nodes later based off of OSM notes and photos I take while surveying.

So, I'll likely give this issue a shot and try to dog-food it locally first. For an example of what a fleshed out region can look like, see:

  * [This area of west Fort Worth](https://www.openstreetmap.org/#map=15/32.73070/-97.46319) (no surveillance highlighted)
    *  ![Image](https://github.com/user-attachments/assets/4f21f30d-2eac-4a7c-971b-56f4b92d38dc)
  * [This overpass-turbo query](https://overpass-turbo.eu/s/26Xf) (red for ALPRs, blue for gunshot detectors, purple when both coincide, yellow with blue ring for other cameras)
    * ![Image](https://github.com/user-attachments/assets/03c2cc82-94f0-4ae8-b860-48e9ab6f065a)
  * The deflock.me website [for the area](https://deflock.me/map#map=16/32.731552/-97.462184) as well (only ALPRs, along with their orientation, highlighted)
    * ![Image](https://github.com/user-attachments/assets/1323ba64-87c6-45f5-8b8b-e9ee8a5eea05)

**Comment by Cj-Malone** [comment]
I've been mapping cameras lately, and I think it is really important to map. Using SC would make mapping them much easier.

I have several questions/concerns abouts specific tags on talk pages, I won't mention them here right now.

The previous time this was disused was in 2020, was that in the overpass era? Does the new query system allow you to get all man_made=surveillance outside of a building geom? That could skip a lot of the issues unanswerable questions.

Direction of camera would be my biggest want at the moment.

**Comment by westnordost** [comment]
> get all man_made=surveillance outside of a building geom

Yes. This is done for the housenumber quest. However, it is computationally expensive, which is why the quest is in the "chonker index" (find the function `getAnalyzePriority` in the repository for an explanation.)

**Comment by riQQ** [comment]
For reference:

https://github.com/streetcomplete/StreetComplete/blob/eb9ae60be07a647c7617459a68fc9a5be6c442cd/app/src/commonMain/kotlin/de/westnordost/streetcomplete/data/osm/osmquests/OsmQuestUtils.kt#L3-L7

https://github.com/streetcomplete/StreetComplete/blob/eb9ae60be07a647c7617459a68fc9a5be6c442cd/app/src/androidMain/kotlin/de/westnordost/streetcomplete/data/osm/osmquests/OsmQuestUtilsAndroid.kt#L15-L24

**Comment by Cj-Malone** [comment]
Presumably the expensive part of the query is only executed if the other parts are true? It would seem all `man_made=surveillance` isn't exactly common right now, 0.5 m, almost all have `surveillance:type=` according to tag info. `surveillance:type=ALPR` and `surveillance=indoor` could be used to pre filter ~0.1m of them.

**Comment by westnordost** [comment]
Yes
